### DISCUSSION TITLE
Robots on all fours

### DISCUSSION THREAD

**Discussion by omichel** [cyberbotics/wrestling#23] [Announcements]
We recently saw that some robots, including Chrake Peith, are getting on all fours after walking around.
This is an excellent strategy as it is a very robust position in which the robot is very unlikely to fall down or fall off the ring.
However, it doesn't look very nice to see robots adopting this turtle position during a game.
This probably won't make the competition evolve towards more intelligent robots, which is our ultimate goal.
Therefore, I would suggest to implement a change in the rules to prevent this type of behavior.
A solution could be to rise the threshold under which a robot considered down.
But the normal standing position of the robot is already 0.833 and the threshold is 0.75, which leaves little margin.
A better solution could be to consider the position of the head of the robot (instead of the body) and set its threshold accordingly to ensure the head position correspond to a standing position.
This second solution appears more robust.
So, unless anyone opposes to [this change](https://github.com/cyberbotics/wrestling/commit/cdec2c153371a9d53468c5c691bad3da539844d0) already in production, we are going to keep it from now on.
Please be sure to update your local copy, if any, to take into account this rule change.

**Comment by pemile29** [comment]
It was after reading that in the absence of a KO the coverage criterion was determined that I found Chrake Peith's approach interesting as a way of getting started with these NAOs, right?

In a second step, studying the examples of motions files will offer us two opportunities of model, a robot which rushes to seek the fight but also a robot expert in escape and avoidance?

**Reply by pemile29** [reply]
_"We recently saw that some robots, including Chrake Peith, are getting on all fours after walking around."_

I missed this demonstration which personally seems unattainable at my level, but at this level of performance no countermeasure would be possible for a nao, no possibility to "lift" an opponent?

**Reply by omichel** [reply]
Yes, I believe it might possible, but difficult, to lift an opponent. Anyhow, humanoid robots are meant to walk and stay on two legs and considering the head of the robot as the part that has to stand above a threshold during the game is very reasonable to me.
